cmd/expired-authz-purger/main.go

```diff
@@ -117,6 +126,46 @@ func (p *expiredAuthzPurger) getWork(work chan string, query string, initialID s
 	return lastID, count, nil
 }
 
+// deleteAuthorizations reads from the work channel and deletes each authorization
+// from either the pendingAuthorization or authz tables. If maxDPS is more than 0
+// it will throttle the number of DELETE statements it generates to the passed rate.
+func (p *expiredAuthzPurger) deleteAuthorizations(work chan string, maxDPS int, parallelism int, table string, checkpointFile string) {
+	wg := new(sync.WaitGroup)
+	deleted := int64(0)
+	var ticker *time.Ticker
+	if maxDPS > 0 {
+		minDur := time.Duration((1/(float64(maxDPS)/float64(parallelism)))*1000) * time.Millisecond
```

Comment:
Let's flip the denominator here: parallelism/maxDPS. Also, no reason to multiple by 1000 and then by Millisecond; just multiply by Second.

Reply:
`time.Duration` is a int, so it truncates floats and makes this not work.

Reply:
(the millisecond -> second thing that is, denom change is fine)